helm/chart-operator-chart/templates/rbac.yaml

```diff
@@ -12,6 +12,14 @@ rules:
   - customresourcedefinitions
   verbs:
   - "*"
+  - apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions/status
+  verbs:
+  - create
```

Comment:
Only 3 verbs are sufficient at now?

Reply:
Yes the CR status should only be created or updated and not deleted. As that could lead to data loss.

In the operators we also try to use `UpdateStatus` rather than `Update`.

e.g. https://github.com/giantswarm/chart-operator/blob/a571ec7a704a3935d0febcffee555e29367eaee5/service/controller/chart/v1/resource/status/create.go#L65